client/src/containers/ChatBot/index.js

```diff
@@ -169,6 +172,4 @@ ChatbotContainer.propTypes = {
   dispatch: PropTypes.func.isRequired,
 };
 
-const ChatbotWithRouter = withRouter(ChatbotContainer);
-
-export default connect()(ChatbotWithRouter);
```

Comment:
That was a very nasty error that disabled one of the very important functionalities :(
This kind of things is very difficult to detect sometimes and they are not covered by tests...